tools/add-badges.py

```diff
@@ -0,0 +1,71 @@
+import re
+import os
+import subprocess
+import fileinput
+import sys
+import shutil
+
+for repo_list_line in open('repositories.txt'):
+    m = re.search('(\d+)\|github.com/OCA/(.*)', repo_list_line)
+    repo_name = m.group(2)
+    repo_id = m.group(1)
+    print(repo_name)
+    if not os.path.exists(repo_name):
+        subprocess.call(['hub', 'clone', '--quiet', 'OCA/' + repo_name])
+
+    os.chdir(repo_name)
+    for version in ['6.1', '7.0', '8.0']:
+        try:
+            subprocess.check_call(['git', 'checkout', '--quiet', version])
+        except subprocess.CalledProcessError:
+            continue
+
+        subprocess.check_call(['git', 'reset', '--hard'])
+        subprocess.check_call(['git', 'clean', '-fxd'])
+        new_lines = (
+            "[![Runbot Build Status]"
+            "(http://runbot.odoo-community.org/runbot/badge/flat/{0}/{1}.svg)]"
```

Comment:
One URL for the build status (svg) and one for the linked page which had all the branches and PRs